File: TOMIMSV4/Entity/Transacciones/OrdenCompra/Embarcador/clsBeTrans_oc_embarcador.vb
Public Class clsBeTrans_oc_embarcador
    Implements ICloneable

    Public Property IdEmbarcador() As Integer = 0
    Public Property Codigo() As String = ""
    Public Property Nombre() As String = ""

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Trans_oc/clsBeTrans_oc_embarcador.cs

namespace WMS.EntityCore.Trans_oc
{
    public class clsBeTrans_oc_embarcador : ICloneable
    {
        public int IdEmbarcador { get; set; } = 0;
        public string Codigo { get; set; } = "";
        public string Nombre { get; set; } = "";

        public clsBeTrans_oc_embarcador()
        {
            // EJC: Add your constructor here...
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
